Add instructions on resolving CRL issues in C# SDK (#2683)
MicrosoftDocs/dataexplorer-docs: data-explorer/security-network-private-endpoint-troubleshoot.md

A successful result must return **Pong!** and an IPv6 address.

> ### Check Remote Cretificate Revocation List 
> 
> If you get an exception saying the remote certificate can not be validated when using the C# SDK, your network may be configured to block checking the remote certificate revocation list.

Reviewer: ```suggestion
### Check remote certificate revocation list 

If you get an exception saying the remote certificate cannot be validated when using the C# SDK, your network may be configured to block checking the remote certificate revocation list.
```

```
System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception. --->
System.Security.Authentication.AuthenticationException: The remote certificate is invalid because of errors in the certificate chain: RevocationStatusUnknown
```

> You can reolve this issue by allowing access the [list of remote certificatre verification uris](/entra/global-secure-access/how-to-configure-connectors#allow-access-to-urls) from the [azure private network configuration guide](/entra/global-secure-access/how-to-configure-connectors).

Reviewer: ```suggestion
You can resolve this issue by allowing access to the [list of remote certificate verification URLs](/entra/global-secure-access/how-to-configure-connectors#allow-access-to-urls) from the [Azure private network configuration guide](/entra/global-secure-access/how-to-configure-connectors).
```

> Alternatively (less recommanded), you can disable the remote certificate revocation validation using one of the following options:

Reviewer: ```suggestion
Alternatively (less recommended), you can disable the remote certificate revocation validation by using one of the following options:
```

> * Using Command Line Args: Add the following arguemnt to your C# process:

Reviewer: ```suggestion
* Using Command Line Args: Add the following argument to your C# process:
```
